SConstruct

```diff
@@ -7,6 +7,11 @@ import time
 import errno
 from SCons.Script import BUILD_TARGETS
 
+try:
+    input = raw_input
```

Comment:
(pylint-redefined-builtin) Redefining built-in 'input'

Reply:
should probably shut this warning up if this is intended

Reply:
Done.